jc/parsers/find.py

```diff
@@ -0,0 +1,121 @@
+"""jc - JSON Convert `find` command output parser
+
+Usage (cli):
+
+    $ find | jc --find
+
+Usage (module):
+
+    import jc
+    result = jc.parse('find', find_command_output)
+
+Schema:
+
+    [
+      {
+        "directory path":           string,
+        "file or empty directory:   string
+      }
+    ]
+
+Examples:
+
+    $ find | jc --find -p
+    [
+        {
+          "directory path": "./directory"
+          "file or empty directory": "filename"
+        },
+        {
+          "directory path": "./anotherdirectory"
+          "file or empty directory": "anotherfile"
+        },
+        ...
+    ]
+
+    $ find | jc --find -p -r
+    {
+      "./templates/readme_template",
+      "./templates/manpage_template",
+      "./.github/workflows/pythonapp.yml",
+      ...
+    }
+"""
+import jc.utils
+
+
+class info():
+    """Provides parser metadata (version, author, etc.)"""
+    version = '1.0'
+    description = '`find` command parser'
+    author = 'Solomon Leang'
+    author_email = 'solomonleang@gmail.com'
+    compatible = ['linux']
+    tags = ['command']
+
+
+__version__ = info.version
+
+
+def _process(proc_data):
+    """
+    Final processing to conform to the schema.
+
+    Parameters:
+
+        proc_data:   (Dictionary) raw structured data to process
+
+    Returns:
+
+        List of Dictionaries. Structured data to conform to the schema.
+    """
+
+    processed = []
+    for i in proc_data:
+        try:
+            temp = str(i).rindex("/")
+        except ValueError:
+            pass
```

Comment:
Think you should set temp above the try/except or within the except so it is assigned even if there is a ValueError. Otherwise you'll get a name error when trying to use temp below.

Maybe an easier way of assigning the path/file would be something like:
```
path, file = i.rsplit('/', maxsplit=1)
```
Then you can just use `path` and `file` as the values below instead of slicing.